### PR TITLE
Resources/us225

### PR DESCRIPTION
**Pull request by tullosjr**
Add Spanish translation to Resources screen.

### CODE REVIEW

File: drawerScreens/glossaryScreens/GlossaryScreenCard.js [hunk 1]
```diff
@@ -4,21 +4,25 @@ import { Card } from "react-native-elements";
...
+    render() {
+        return (
+            <Card
+                featuredTitle={translate("glossary.glossary")}
+                featuredTitleStyle={{ fontSize: 40 }}
+                image={require("../../assets/GlossaryScreen.jpg")}
+                accessible
+                accessibilityLabel={translate("glossary.accessabilityLabel")}
+            >
+                <Text
```
Comment (tullosjr) [inline drawerScreens/glossaryScreens/GlossaryScreenCard.js]: Formatting

File: drawerScreens/glossaryScreens/GlossaryScreenCard.js [hunk 2]
```diff
@@ -4,21 +4,25 @@ import { Card } from "react-native-elements";
...
+                        fontWeight: "500",
+                        textAlign: "center"
+                    }}
+                >
+                    {translate("glossary.description")}
+                </Text>
+            </Card>
+        );
+    }
 }
```
Comment (tullosjr) [inline drawerScreens/glossaryScreens/GlossaryScreenCard.js]: Formatting

File: drawerScreens/glossaryScreens/GlossaryScreen.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+import { Button, Platform, ScrollView, StyleSheet, View } from "react-native";
 import NavigationService from "../../components/NavigationService";
 import PchPtsdAccordion from "../../components/shared/PchPtsdAccordion";
+import translate from "../../components/translateService";
 import GlossaryScreenCard from "./GlossaryScreenCard";
 import GlossaryTerms from "./GlossaryTerms";
-import translate from "../../components/translateService";
 
 class GlossaryScreen extends Component {
     static navigationOptions = {
```
Comment (tullosjr) [inline drawerScreens/glossaryScreens/GlossaryScreen.js]: Organize imports

**Comment by ardecost** [comment]
Code reviewed and tested. Glossary and Resource translations working as intended.

**Pull request merged**